install/tomcat_conf/server_adaptation.xml

```diff
@@ -0,0 +1,17 @@
+<Resource name="jdbc/cws_adaptation"
+          auth="Container"
+          type="javax.sql.DataSource"
+          factory="com.zaxxer.hikari.HikariJNDIFactory"
+          poolName="cws-adapt-pool"
+
+          maximumPoolSize="5"
+          connectionTimeout="300000"
+
+          driverClassName="__CWS_ADAPT_DB_DRIVER__"
+          jdbcUrl="__CWS_ADAPT_DB_URL__"
+          username="__CWS_ADAPT_DB_USERNAME__"
+          password="__CWS_ADAPT_DB_PASSWORD__"
+
+          transactionIsolation="TRANSACTION_READ_COMMITTED" />
```

Comment:
This database is for adaptation (non-camunda tables), and therefore does not have to have the TRANSACTION_READ_COMMITTED level.  This level is a requirement of Camunda, but not a usual requirement of other database purposes.  I would probably just leave this line out.

Reply:
Removed `TRANSACTION_READ_COMMITTED ` line